packages/elements/src/components/button/__styles__/index.tsx

```diff
@@ -113,13 +141,52 @@ export const ElButton = styled.button`
       }
     }
   }
+
+  ${isMobile} {
+    height: 2.5rem;
+    font-size: 1rem;
+    padding: ${buttonYPaddingMobile}rem 0.75rem;
+  }
+
+  &.el-floating-button {
```

Comment:
`.${elFloatingButton} {`

Reply:
If the consuming app hashes the class name, `el-floating-button` won't be a thing!